dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/AcmeProviderImpl.java

```diff
@@ -700,6 +712,14 @@ public File createDefaultSSLCertificate(String filePath, @Sensitive String passw
 			 */
 			File file = createKeyStore(filePath, acmeCertificate, password, keyStoreType, keyStoreProvider);
 
+			/*
+			 * Create the acme file which holds certificate information and a record
+			 * of directoryURIs. We use this to determine if the directoryURI has
+			 * been updated. If so, we need to refresh the certificate.
+			 */
+			createAcmeFile();
```

Comment:
Maybe this can just be done inside update? If it doesn't exist, try and create it?